includes/class-wc-payments-order-service.php

```diff
@@ -518,6 +519,25 @@ public function get_charge_id_for_order( $order ) : string {
 		return $order->get_meta( self::CHARGE_ID_META_KEY, true );
 	}
 
+	/**
+	 * Get any dispute data for specified order.
+	 *
+	 * @param  mixed $order The order Id or order object.
```

Comment:
`nit`

```suggestion
	 * @param mixed $order The order Id or order object.
```

Aligning is necessary only with lines right below or after, right? In this case, there is a new-line.
Not important though as PHP linter does not complain.